insecure-skip-tls-verify is only about VU code (#140)
loadimpact/k6-docs: src/data/markdown/docs/01 guides/02 Using k6/05 Options.md

### Insecure Skip TLS Verify

A boolean, true or false. When this option is enabled (set to true), all of the verifications that
would otherwise be done to establish trust in a server provided TLS certificate will be ignored. 

> This is only about connections created by VU code such as http requests.

Reviewer: ```suggestion
This only applies to connections created by VU code, such as http requests.
```

Reviewer: With that change, I think it will be ready to merge.
